src/dataaccess/utils/fhir-entry-processor.js

```diff
@@ -26,11 +26,16 @@ export default function(responses, patientId, resourceMapper = null) {
     const mappedResources = {};
     const referencesOut = [];
     const allObjects = entries.map(entry => {
+        const resource = entry.resource;
```

Comment:
Is there some sort of default mapping that can be done?  If FHIR Observation instantiate SHR Observation ...

Reply:
We talked about that a little bit some months ago when I initially put this in. I think at the time we decided it was a little duplicative with the FHIR mapper project. I can add something, but I don't think it's really necessary, since a data source "should" either be entirely profiled in which case it's not necessary to do any mapping (ex, via a shim server or a FHIR server containing mCODE'd data) or entirely non-profiled in which case we should configure the full FHIR mapper (ex, via any server that exists today).